### PR TITLE
fetch-demo

### PR DESCRIPTION
**Pull request by DJTechnoo**
Added a demo showing how to manually fetch data without the connector.

### CODE REVIEW

File: demos/manual-fetch/demo.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        const res = await fetch(msURL, morningstarReqPayload);
+
+        if (!res.ok) {
+            throw new Error(`Fetch failed with status: ${res.status}`);
+        }
+
+        resJson = await res.json();
+
+    } catch (error) {
+        // Fail silently
```
Comment (pawellysy) [inline demos/manual-fetch/demo.js:116]: add the console log here, or there is no point in writing error msgs where you throw them.

**Review by pawellysy — APPROVED** [review]
minor polishing apart from that looks good!

File: demos/manual-fetch/demo.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            item => [new Date(item.EndDate).getTime(), Number(item.Value)]
+        );
+
+    // Finally display the fetched data
+    Highcharts.stockChart('container', {
+        title: {
+            text: 'Manually fetched data'
+        },
+        xAxis: {
+            type: 'datetime',
```
Comment (pawelfus) [inline demos/manual-fetch/demo.js:131]: `datetime` is default in Stock, no need to set this explicit

**Review by pawelfus — CHANGES_REQUESTED** [review]
Looks good, just a tiny request

**Review by kamil-musialowski — APPROVED** [review]

**Review by bre1470 — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]

**Pull request merged**